content/merchants/index.md

```diff
@@ -20,6 +20,19 @@ type: "default"
 <p>If your store accepts NavCoin and you would like to be listed on this page, read about how you can contribute to this website.
 {{< /left_image_section >}}
 
+{{< home_centre_text_section
+    title="World map of stores and merchants accepting NAV"
+    linkBgColor="#43799a"
+    gradient=""
+    fontColor="#FFF">}}
+    <div class="merchant-map">
+      <iframe src="https://www.google.com/maps/d/embed?mid=1wzMJAyujqSQ-Iyy0mqQxP_uuwrphIuYJ&hl=en" width="720" height="540"></iframe>
```

Comment:
You should be able to remove the `width` and `height` here and do it in the style and percentages and `max-width` / `max-height` to allow working on mobile. (Untested)